test/replaytests/workflows.go

```diff
@@ -261,3 +262,68 @@ func VersionLoopWorkflow(ctx workflow.Context, changeID string, iterations int)
 	}
 	return workflow.Sleep(ctx, time.Second)
 }
+
+func ChildWorkflowWaitOnSignal(ctx workflow.Context) error {
+	workflow.GetSignalChannel(ctx, "unblock").Receive(ctx, nil)
+	return nil
+}
+
+func DuplicateChildWorkflow(ctx workflow.Context) error {
```

Comment:
This workflow will replay differently before and after my change, hence the use of SDK flags.